src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -41,7 +41,7 @@
 
     <!-- refresh button -->
     <div class="widget-toolbar tc_refreshBtn">
-      <a (click)="reloadData()">
```

Comment:
Could you revert this? autoReload and clicking the btn should do the same thing.

Reply:
Than the reload if you click is not possible without reloading the same data for example in mirroring for 5 times! Each time the auto reload is triggered. Also in the end they are doing the same thing the difference is that the user clicks on it and want's feedback (loading indicator). If the table auto reloads every 5 seconds the user will be distracted by the loading indicator flashing up the hole time.